File: swagger/models/remove_from_waitlist_request.py
from .base_model import BaseModel


class RemoveFromWaitlistRequest(BaseModel):
	"""
	no description available

	Attributes:

		waitlist_entry_ids
		A list of waiting list IDs to remove from waiting lists.
	"""

	input_map = {
		'WaitlistEntryIds': 'waitlist_entry_ids',
		}

	output_map = {
		'waitlist_entry_ids': 'WaitlistEntryIds',
		}	def __init__(self, waitlist_entry_ids):
		self.waitlist_entry_ids = waitlist_entry_ids

File: swagger/Models/RemoveFromWaitlistRequest.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class RemoveFromWaitlistRequest extends BaseModel
{
	/**
	 * A list of waiting list IDs to remove from waiting lists.
	 * 
	 * @var integer[] 
	 */
	public $waitlistEntryIds;
	/**
	 * Constructor
	 */
	public function __construct($waitlistEntryIds)
	{
		$this->waitlistEntryIds = $waitlistEntryIds;
	}

	protected function getInputMap()
	{
		return [
			'WaitlistEntryIds' => ['waitlistEntryIds', 'array', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'waitlistEntryIds' => ['WaitlistEntryIds', 'array', null],
			];
	}
}
